tools/portlet_spring_mvc_tmpl/ivy.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+
+<ivy-module
+	version="2.0"
+	xmlns:m2="http://ant.apache.org/ivy/maven"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="http://ant.apache.org/ivy/schemas/ivy.xsd"
+>
+	<info module="@portlet.name@-portlet" organisation="com.liferay">
+		<extends extendType="configurations,description,info" location="${sdk.dir}/ivy.xml" module="com.liferay.sdk" organisation="com.liferay" revision="latest.integration" />
+	</info>
+
+	<dependencies defaultconf="default">
+		<dependency org="org.springframework" name="spring-webmvc" rev="3.0.7.RELEASE"/>
+		<dependency org="org.springframework" name="spring-webmvc-portlet" rev="3.0.7.RELEASE"/>
+		<dependency conf="test->default" name="arquillian-junit-container" org="org.jboss.arquillian.junit" rev="1.1.3.Final" />
```

Comment:
Contrary all other template do have the tests included in master. That's the reason I left them there.